src/test/java/jpa/AuthorTest.java

```diff
@@ -0,0 +1,58 @@
+package jpa;
```

Comment:
This file is already in the master, this branch may need to be updated before approval. As there might be conflict with #53 since that changes some method names.